### PR TITLE
Minor cleanup

### PR DESCRIPTION
**Pull request by sergeyklay**

### CODE REVIEW

File: php-mode.el [hunk 1]
```diff
@@ -24,7 +24,7 @@
 
 ;; This file is distributed in the hope that it will be useful,
 ;; but WITHOUT ANY WARRANTY; without even the implied warranty of
-;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
```
Comment (zonuexe) [inline php-mode.el]: Do not remove two spaces after period.
Please use [purcell/flycheck-package: Flycheck checker for elisp package metadata](https://github.com/purcell/flycheck-package).

**Comment by ejmr** [comment]
Thanks for the pull request and patch.  Two comments though:

1. Please write a more descriptive commit message.  *"Update php-mode.el"* could mean literally anything.  Ideally a person should never be forced to look at the code to see exactly what a patch does, because the commit message should always describe that in detail.

1.a.  If a patch fixes typos, adds or updates documentation, corrects information in tr he README---and so on---then the commit message needs to make that clear.  Basically the commit message needs to say something along the lines of, *"This patch only fixes a few typos and does not affect any behavior in PHP Mode."*

2. Thank you for fixing the "mode" typos.  But please restrict the patch to only those typos.  Or to put it another way, do not change sentences from using two-spaces after the period into sentences which only use one space.  And do not change the indentation of code which the patch does not affect.  The same goes for adding more blank lines.  The problem with doing this is that in the future developers could easily get the wrong idea and think there was a meaningful change in a piece of code where in reality all that changed was the indentation.  It saves time and headaches in the future.

But other than that we should definitely use "PHP Mode" consistently with *Mode* capitalized.  If you could please restrict the patch to doing only that then I will definitely merge it.

File: php-mode.el (hunk 1 above)
Comment (ejmr) [inline php-mode.el]: @zonuexe You beat me to it, lol.

**Comment by sergeyklay** [comment]
Close in favor of #369

**Pull request closed without merging**